test/JDBC/expected/BABEL-492.out

```diff
@@ -12,39 +12,29 @@ CREATE PROC p492 @a INT, @b FLOAT AS
 SELECT @a, @b;
 GO
 
-CREATE PROC p492 @a INT, @b DECIMAL AS
```

Comment:
this test has now two variants. One via upgrade where we will still get the error. Other is for latest which is no longer valid. We should keep the upgrade variant at-least.